### PR TITLE
Text utilities and modal

### PR DESCRIPTION
**Pull request by Nagashree01**
Added Text Utilities

- headings
- small text
- gray text
- center text

Added Modal
modal

### CODE REVIEW

File: doc.css [hunk 1]
```diff
@@ -22,78 +22,85 @@
...
+.badge-container .button-container {
     padding: 1rem;
 }
 
-.avatar-component-view{
-    padding:1rem 2rem;
+.avatar-component-view {
+    padding: 1rem 2rem;
     display: flex;
     flex-direction: row;
```
Comment (sushil-mod) [inline doc.css:68]: i think you can remove this as default is row for flex

File: doc.css [hunk 2]
```diff
@@ -22,78 +22,85 @@
...
     gap: 3rem;
 }
 
-.alert-component-view{
-    display:flex;
+.alert-component-view {
+    display: flex;
     flex-direction: column;
```
Comment (sushil-mod) [inline doc.css:74]: here

**Review by sushil-mod — APPROVED** [review]
just asuggestion , modal functionality working fine , overall code is good .

File: doc.css (hunk 1 above)
Comment (ultronjs) [reply doc.css:68]: Same as Sushil

File: documentation.html [hunk 3]
```diff
@@ -517,12 +517,72 @@ <h1 id="grid-comp" class="component-name">Grid</h1>
...
+                </div>
+            </div>
+            <!-- MODAL -->
+            <div class="modal-component-container">
+                <h1 id="modal-comp" class="component-name">Modal</h1>
+                <p class="component-info"></p>
+                <div class="component-container-view ">
+                    <div><button class="showModalBtn btn btn-primary">Show Modal</button></div>
+                    <div class="modal-wrapper">
+                        <div class="modal flex column">
```
Comment (ultronjs) [inline documentation.html:560]: No need for this div. You can add flex column class to the above div also.It will work fine there also

**Review by ultronjs — APPROVED** [review]
The code is clean + modular.
Naming could be better at some of the places like heading and flex utility.
Mentioned Some comments. You can go through that.
Otherwise Good to merge.

**Review by Manju-sree — APPROVED** [review]
Added few suggestions, I think it would help you, bcoz these changes would give you better marks later. so, try to make changes if possible. otherwise this PR looks fine to merge.

File: components/typography.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.pg-lg{
+font-size: 1.5rem;
+}
+
+.pg-md{
+    font-size: 1rem;
+}
+.pg-sm{
```
[the remark this answers is not in the document]
Comment (Nagashree01) [reply components/typography.css:35]: yes will use better naming convention and move it to utility class. Thank you

[the remark this answers is not in the document]
Comment (Nagashree01) [reply components/typography.css:35]: yes will use better naming convention and move it to utility class. Thank you

File: documentation.html (hunk 3 above)
Comment (Nagashree01) [reply documentation.html:560]: yes will remove this

Comment (Nagashree01) [reply documentation.html:560]: yes will remove this

File: components/typography.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
+@import url('/stylesheets/base.css');
+
+.heading-1 {
```
[the remark this answers is not in the document]
Comment (Nagashree01) [reply components/typography.css:3]: yes will give  better class name

[the remark this answers is not in the document]
Comment (Nagashree01) [reply components/typography.css:3]: yes will give  better class name

**Pull request merged**